tensorflow/lite/tools/cmake/modules/Findgoogletest.cmake

```diff
@@ -0,0 +1,29 @@
+#
+# Copyright 2021 The TensorFlow Authors. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+if(googletest_POPULATED)
+  return()
+endif()
+
+include(FetchContent)
+
+FetchContent_Declare(
```

Comment:
@terryheo 
Should we always use OverridableFetchContent_Declare()?

Reply:
I prefer to use it for the consistency.

Reply:
Ok, I will turn to use OverridableFetchContent_Declare() in next commit.

Reply:
@terryheo 
This is the commit using OverridableFetchContent_Declare().
9afaf4824411c1ad3ea9730e318b82e49b6cd307